images/openwisp_nginx/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM nginx:1.23.4-alpine
+FROM nginx:1.27.0-alpine
 
 RUN apk add --update --no-cache \
-            openssl~=3.0.15-r0 \
```

Comment:
openssl is included in the base image. 

https://github.com/nginxinc/docker-nginx/blob/239684923b2c652b0767540d180de7f7e84bd9fa/mainline/alpine-slim/Dockerfile#L24